app/src/pages/header/components/AddMovieForm.Connect.jsx

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+import { AddMovieForm } from './AddMovieForm';
+import { addMovie } from '@store/actionCreators';
+
+let mapDispatchToProps = (dispatch) => {
+    return {
+        addMovie: movie => dispatch(addMovie(movie)),
+    }
+}
+
+export default connect(null, mapDispatchToProps)(AddMovieForm)
```

Comment:
it's ok and u can leave as is, but redux advice to use such approach for mapDispatchToProps:

`export default connect(null, { addMovie })(AddMovieForm)`

Reply:
Done)